torch/csrc/distributed/rpc/python_functions.cpp

```diff
@@ -22,6 +22,32 @@ namespace rpc {
 
 namespace {
 
+c10::intrusive_ptr<c10::ivalue::Future> wrapFutureMessageInJitFuture(
+    const std::shared_ptr<FutureMessage>& responseMessageFuture) {
+  // Notice, even we can ask for the JIT type of the Python object,
```

Comment:
Is this he reason that the code below always uses `PyObjectType::get()` instead of getting the real ivalue type? I am trying to understand the implication here. If a builtin operator returns a Tensor, but we mark it as a `py::object`, are we going to encounter problems if we later feed that tensor as a parameter into another builtin operator?

Reply:
Because after calling, future.wait(), the ivalue::Future layer is shedded. User gets a Python object that is a torch.Tensor(). So the ivalue::Future.type does not matter here.
User can still pass the Python object to the torch.Tensor() to a next builtin call.